src/katgpucbf/fgpu/main.py

```diff
@@ -125,10 +127,17 @@ def _parse_stream(value: str, kws: _OD, field_callback: Callable[[_OD, str, str]
                         kws[key] = float(data)
                     case "dst":
                         kws[key] = endpoint_list_parser(DEFAULT_PORT)(data)
+                    case "dither":
+                        try:
+                            kws[key] = bool_map[data.lower()]
+                        except KeyError:
+                            raise ValueError("dither must be set to 'true' or 'false'") from None
```

Comment:
Must admit, maybe the first time I'm seeing `raise ... from ...`
* I gather it's because you want to treat the `KeyError` as a `ValueError`, actually
* You do give it a default on Line 140, but this is more to ensure the user gives a proper/explicit instruction?

Reply:
`raise ... from None` suppresses the KeyError in the traceback. Without it, Python would tell you about the KeyError (with traceback), then it would say that while processing the KeyError, another exception occurred (the ValueError) and you get that traceback.

Overall this is to ensure you get a helpful message if you try to set it to something that is neither true nor false.

Reply:
Indeed indeed, you worded it a lot better. Still, really neat.